src/backend/TrafficCourts/Citizen.Service/Validators/Rules/CountSectionRule.cs

```diff
@@ -22,13 +22,17 @@ public CountSectionRule(Field field, Services.ILookupService lookupService) : ba
 
     public override void Run()
     {
-        string? value = Field.Value;
-        if (!String.IsNullOrEmpty(value))
+        if (!String.IsNullOrEmpty(Field.Value))
         {
-            value = Field.Value = Regex.Replace(value, @"\s+", "");
-            IEnumerable<Statute> statutes = _lookupService.GetStatutes(value.ToLower().Trim());
-            if (statutes is null || !statutes.Any()) {
-                AddValidationError(String.Format(ValidationMessages.CountSectionInvalid, Field.Value));
+            Field.Value = Regex.Replace(Field.Value, @"\s+", "");

```

Comment:
would assist the reader to add a comment at end of line for each of these...

```cs
Field.Value = Regex.Replace(Field.Value, @"\s+", ""); // remove any whitespace
Field.Value = Regex.Replace(Field.Value, @"^\$$", ""); // remove leading $

Reply:
Done.